arrow-libs/optics/arrow-optics-reflect/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+  id(libs.plugins.kotlin.multiplatform.get().pluginId)
+  alias(libs.plugins.arrowGradleConfig.kotlin)
+  alias(libs.plugins.arrowGradleConfig.publish)
+}
+
+apply(plugin = "io.kotest.multiplatform")
+
+apply(from = property("TEST_COVERAGE"))
+apply(from = property("ANIMALSNIFFER_MPP"))
+
+kotlin {
+  sourceSets {
+    jvmMain {
+      dependencies {
+        api(projects.arrowCore)
+        api(projects.arrowOptics)
+        api(libs.kotlin.stdlibCommon)
+        implementation(libs.kotlin.stdlibJDK8)
```

Comment:
So this exposes the API of Kotlin Std, and uses the JDK8 implementation?

I'm still not a 100% sure how this works, and what is the correct config :/ 
I don't we're doing different things in different modules.. but off-topic for this PR.

Reply:
I'm not sure 100% either. In any case, I've removed the `api` dependency on `stdlibCommon` and just left the `implementation` for the JDK version.